packages/algob/src/internal/cli/unbox-template.ts

```diff
@@ -0,0 +1,188 @@
+import chalk from "chalk";
+import enquirer from "enquirer";
+import fse from "fs-extra";
+import path from "path";
+
+import { copyTemplatetoDestination, fetchRepository, setUpTempDirectory } from "../util/unpack";
+import { createConfirmationPrompt, installDependencies, printSuggestedAlgobCommands, printWelcomeMessage } from "./project-creation";
+
+const ALGOB_DAPP_TEMPLATES_GIT_REMOTE = 'scale-it/algorand-builder-templates';
```

Comment:
please rename this to `algo`

Reply:
`algob`? I renamed `'scale-it/algorand-builder-templates'` -> `'scale-it/algo-builder-templates'`